docs/manual/java/gettingstarted/GettingStartedMaven.md

```diff
@@ -67,18 +67,18 @@ $ mvn lagom:runAll
 (Services started, press enter to stop and go back to the console...)
 ```
 
-You can verify that the services are indeed up and running by invoking one of its endpoints, e.g:
+You can verify that the services are indeed up and running by invoking one of its endpoints from any HTTP client. For example, on Windows you might use Postman. On Linux, you might use curl:
```

Comment:
hmm, that was my intention by saying "from any HTTP client", but your suggestion is more clear. How do I change this without creating another branch?